src/pages/changelog/post-15.mdx

```diff
@@ -0,0 +1,93 @@
+
+## Manage your Data
+
+### Exporting Data
+
+Core to the idea of a personal cloud computer is the ability to have control over your data. This also includes being able to export your data from Space to use for what ever you want.
+
+<Image src="/changelog_assets/export-data.png" alt="Export Data" width="90%" background />
+
+You can now export your data from the "Data" view of your app instance. This will export all Bases and Drives that your app uses to a ZIP file that you can download.
+
+### Clear App Data
+
+Want to start fresh? You can now clear all the data in your app instance. This will delete all the Bases and Drives that your app uses. You can do this from the "Data" view of your app instance as well.
+
+<Image src="/changelog_assets/clear-app-data.png" alt="Clear App Data" width="90%" background />
+
+### Delete App, Keep Data
+
+Sometimes, you want the opposite of wiping the data, which is deleting the app, but keeping the data. Sounds wild, right? Dump the app, but cherish the data in a collection.
+
+<Image src="/changelog_assets/delete-app-keep-data.png" alt="Delete App" width="90%" background />
+
+## Managing your Project
+
+### Delete Releases
+
+Ever released something by accident or want to delete an old release? You can now do that from the "Publish" tab in your app's dashboard.
+
+<Image src="/changelog_assets/manage-release.png" alt="Manage Release" width="90%" background />
+
+If the release doesn't have any installations yet, you can delete it. If it does, you can unlist it from Discovery instead. This will remove it from the app's page and the Discovery homepage. Existing installations will still be able to use it.
+
+### Delete Project
+
+You are now able to delete your entire project. This will delete all the releases and installations of your app. If your app is listed on Discovery and has existing installations, you won't be able to delete it for now. We are working on a way to do this safely.
+
+<Image src="/changelog_assets/unlist-project.png" alt="Unlist Project" width="90%" background />
+
+### Unlist Project from Discovery
+
+If you want to hide your entire app from Discovery instead, you can do that from the same menu. This will remove the app and all releases from being listed publicly on Discovery. Existing installations will continue working.
+
+## Environment Variables and the CLI
+
+You can now use the `space builder env` command of the [Space CLI](https://deta.space/docs/en/build/reference/cli) to fetch or update the environment variables in your project's development instance.
+
+Fetching them:
+
+```bash
+space builder env --get .env
+```
+
+Update them:
+
+```bash
+space builder env --set .env
+```
+
+Note that this is completely decoupled from your local [Spacefile](/docs/en/reference/spacefile/), and you need to push a new development instance if you add or remove a custom environment variables.
```

Comment:
What does this mean, push a new dev instance if i add / remove custom env vars?

Reply:
Took this from [the docs for the command](https://github.com/deta/space-docs/pull/162/files). I think it refers to adding a new custom environment variable in which case you have to specify it in the Spacefile. Maybe @brkp can expand on it

Reply:
so if you add a new custom env variable to your Spacefile, you need to push your application once before you can start using that variable with `space builder env`. this is the case because it's not trivial to add new env variables to the lambdas